### ISSUE TITLE
The configfile is world readable

### ISSUE THREAD

**Issue by quartje** [eduvpn/python-eduvpn-client#315]
The config is downloaded in ~/config/eduvpn, but with 644 mode of access, which means it's world readable. 600 should be sufficient I guess.
The config directory itself is created with access mode 755. That should be 700

**Comment by gijzelaerr** [comment]
good point. which version are you currently using? I thought we handled that properly in 1.x

**Comment by gijzelaerr** [comment]
interesting, that should return 1.1, which is another bug.
